src/components/Project/ProjectContent/ContentView.vue

```diff
@@ -3,10 +3,11 @@
     <div class="tabs-wrapper">
       <TabView
         class="tab-view-wrapper"
+        :class="{ draggable: focus }"
         :active-index="activeIndex"
         @tab-click="onTabClick($event)"
       >
-        <TabPanel v-for="(tab, i) in tabs" :key="tab.id">
+        <TabPanel v-for="(tab, i) in tabs" :key="tab.id" class="ankit-vats">
```

Comment:
do not use any developer name, change it's name logically

Reply:
Used this for testing, forgot to remove it. Already removed in the next PR.